.github/workflows/ci.yml

```diff
@@ -1,18 +1,28 @@
 name: CI
+
 on:
   push:
     branches: [master]
+    paths:
+      - grammar.js
+      - src/**
+      - bindings/**
+      - binding.gyp
   pull_request:
-    branches: [master]
+    paths:
+      - grammar.js
+      - src/**
+      - bindings/**
+      - binding.gyp
+
 jobs:
   build:
+    name: Run tests
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: tree-sitter-grammars/actions/setup@main
+        with:
+          node-version: 20
+      - uses: tree-sitter-grammars/actions/test@main
```

Comment:
```suggestion
      - uses: tree-sitter/parser-test-action@v1.2
```

Reply:
should this be fixed in the the tree-sitter-grammars action repo? I just copied from there

Reply:
I archived it and linked to upstream.